source/jormungandr/jormungandr/parking_space_availability/bss/atos.py

```diff
@@ -63,23 +63,39 @@ def support_poi(self, poi):
         return properties.get('operator', '').lower() in self.operators and \
                properties.get('network', '').lower() == self.network
 
+    def get_navitia_status(self, status):
+        if status == 'CONNECTEE':
+            return 'Open'
+        else:
+            return 'Unavailable'
+
     def get_informations(self, poi):
         logging.debug('building stands')
         try:
+            # Possible status values of the station: CONNECTEE and DECONNECTEE
             all_stands = self.breaker.call(self._get_all_stands)
             ref = poi.get('properties', {}).get('ref')
-            if ref:
-                stands = all_stands.get(ref.lstrip('0'))
+            if not ref:
+                return Stands(0, 0, 'Unavailable')
+            stands = all_stands.get(ref.lstrip('0'))
+            if stands:
+                if stands.status in ['Closed', 'Unavailable']:
```

Comment:
'Closed' isn't a possibility here. Either add it in the navitia status or it's useless to check it here.